lib/verdict/experiment.rb

```diff
@@ -268,12 +280,16 @@ def nil_assignment(subject)
   private
 
   def is_scheduled?
-    if @schedule_start_timestamp and @schedule_start_timestamp > Time.now
+    if @schedule_start_timestamp && @schedule_start_timestamp > Time.now
```

Comment:
making this change because this [post](https://stackoverflow.com/questions/27429711/difference-between-not-and-in-ruby)

Reply:
Was this broken because of the old precedence then?

Reply:
this wasn't broken but I thought it might be good practice to always use the highest precedence in this case. What do you think?

Reply:
Makes sense.